debian/packetfence-config.postinst

```diff
@@ -29,10 +35,7 @@ case "$1" in
             echo "pfconfig.conf already exists, won't touch it!"
         fi
 
-        if [ ! -f /usr/local/pf/conf/system_init_key ]; then
-            hexdump -e '/1 "%x"' < /dev/urandom | head -c 32 > /usr/local/pf/conf/system_init_key
-        fi
-
+        setup_system_init_key
```

Comment:
So, this is the earliest place to put in, correct?
I recall that you said it needs to be as soon as possible.